junit-platform-commons/src/main/java/org/junit/platform/commons/util/BlacklistedExceptions.java

```diff
@@ -36,26 +33,26 @@
  *
  * @since 1.0
  */
-@API(status = INTERNAL, since = "1.0")
+@API(status = DEPRECATED, since = "1.7")
 public final class BlacklistedExceptions {
```

Comment:
Please add an `@deprecated` Javadoc tag at the class level analogous to the one at the method level.

Reply:
Added.